rllib/core/models/configs.py

```diff
@@ -190,6 +190,9 @@ class CNNEncoderConfig(ModelConfig):
 
     @_framework_implemented(tf2=False)
     def build(self, framework: str = "torch") -> Model:
+        self.output_dim = int(self.output_dim)
+        self.input_dims = [int(i) for i in self.input_dims]
```

Comment:
@kouroshHakha Somehow forgot these in the other PR.
We can do asserts here if you like, too. But this is mainly because we get numpy arrays from further up that don't match the operations of the lower level models that expect ints.